General updates related to RooFit warning, Restructuring workspaces path, etc.

ram1123 opened the pull request:
1. Remove all print command statements, instead move it inside the [“processCommand” function](https://github.com/ram1123/Fiducial_XS/blob/de5235d89e00e5b4a75b477eb5166aa8824232d3/runHZZFiducialXS.py#L88-L93). So that the command will be printed before its execution.2.  2. 
   - This reduces many print statements as visible in from diff in current PR.
3. Removed hardcoded model name. Instead added it as the one of option of script “runHZZFiducialXS.py”.
    - This is the way this script was designed earlier. But, currently this option was not used       
    - I added [here](https://github.com/ram1123/Fiducial_XS/blob/de5235d89e00e5b4a75b477eb5166aa8824232d3/runHZZFiducialXS.py#L43) and removed the hardcoded instance from [here](https://github.com/vukasinmilosevic/Fiducial_XS/blob/d6b914652262f7cf0aff97ac4025f92c7ab28b06/runHZZFiducialXS.py#L779)
4. Removed the option `Silence()` from RooFit. We will get rid of silence warnings.
    - This is known issue. It is fixed in ROOT 6.18. 
    - Details: https://root-forum.cern.ch/t/rooworkspace-import-roofit-silence-does-not-work-when-importing-datasets/32591/2
5. Deleted directory `xs_125.0_1bin`. Also, checked after deleting this. Each step is running fine.
6. About many other combine warnings, an issue is created with details: https://github.com/vukasinmilosevic/Fiducial_XS/issues/12

### To-do

- The boundary warning coming from combine. Will try in next PR.

ram1123 committed:
Addded .gitignore file and indentation update done automatically by my editor

ram1123 committed:
update

ram1123 committed:
cleaned: Started by deleting everying then added each file one by one depending on error message

ram1123 committed:
Updating as per Tahir PR vukasinmilosevic/Fiducial_XS#1

ram1123 committed:
Created a model directory and added HZZ model into that

ram1123 committed:
Fix the bug introduced from optparser in efficiencyFactor.py

ram1123 committed:
Updated all main changes from Tahir main PR vukasinmilosevic/Fiducial_XS#1

ram1123 committed:
Restructured directory

ram1123 committed:
Update

ram1123 committed:
Update

ram1123 committed:
Fix the two changes as discussed:

1. Name change of the directory for combine output: `combineLogs` -> `combineOutputs`
2. Uncomment the commented part that I did by mistake

ram1123 committed:
- Bug fix
- Update print statements

ram1123 committed:
Redirected all outputs to the directory xs_125.0

ram1123 committed:
Update

ram1123 committed:
- Fix merge conflict
- Update paths of root files

ram1123 committed:
Minor print changes and added docstring for few functions

ram1123 committed:
- Make uniform convention for directory for combine related .root files
- In earlier versin it was mixed up between `xs_125.0` and `combineOutputs`
- Now, all the hardcoded instance of xs_125.0 is removed and instead used a variable named `combineOutputs` and this variable store `xs_125.0` in every file whereever its required.

ram1123 committed:
moved the  from RunEverything.py to efficiencyFactors.py

ram1123 committed:
Removed all hardcoded instances of directory `datacardInputs` instead added variable `datacardInputs`.

ram1123 committed:
Removed the doLScan_mass4l.sh instead added all 8 commands inside runHZZFiducialXS.py. Don't think we have some benefiting in keeping doLScan_mass4l.sh

ram1123 committed:
Defined a file  and added output directories name here. And linked this to all relevant files

vukasinmilosevic reviewed (approved):
All looks good from my side

ram1123 committed:
Understanding efficiency factors (#1)

* - BugFix: Angular variables (i.e. phi, phi1, etc) was giving error as reported in vukasinmilosevic/Fiducial_XS#17
- Exported obsList.py to ObsList.yml

* Update the modules load method

- Earlier I was adding (using sys.path.insert()) the paths where modules where present
- Now, I added a `setup.sh` script to put these path where modules are present to the PYTHONPATH
- Using this I won't have to bother about the exact location where these modules are present and load in each macro with respect to the relative path
- Earlier, if I change the name of main directory from "Fiducial_XS" to somethine else then few macros won't able to load.

* Don't remove the executable main_fiducialXSTemplates everytime; it will be updated if we updated the main_fiducialXSTemplates.cpp using make command

* Now, plot2dsigeffs.py is importing label name from YAML file

* Added unit in the observabel YAML file, its used in the plotDifferentialBins.py; Now, removed many unit related line from plotDifferentialBins.py, as its taken from YAML file now

* - moved back the ROOT import where it was earlier because of optparser and ROOT options interface.
- Reference: https://root-forum.cern.ch/t/python-options-and-root-options/4641/3

* removed __main__ from efficiencyFactors.py

ram1123 committed:
Added GEN into in observables_list.yml file

ram1123 commented:
Hi @vukasinmilosevic ,

Somehow, I missed the 

```python
import yaml
```

from the file [plotDifferentialBins.py#L7](https://github.com/vukasinmilosevic/Fiducial_XS/blob/ae5a818c843e86a171fdfe95bbe47ded431163ab/python/plotDifferentialBins.py#L7)

I was going to check the interpolation module and before that I first did the `git status` and realised this part I forgot to commit.

Can you include this in your latest PR? Or should I create new one for this?

vukasinmilosevic commented:
Added as a part of #19
